README.md

```diff
@@ -148,6 +151,27 @@ CSS for your templates must be included separately.
 
 See dist/example-config.js for existing features
 
+## External API
+*Currently this API is fluid and should be considered unstable, though we will attempt to adhere to semver guidelines during ongoing development*
+
+When creating an instance of Mappy from config, the return value is an object with a defined interface
+
+Current API:
+```js
+showGeojsonLayer('layerName')
+hideGeojsonLayer('layerName')
+
+stopDataService('dataServiceName')
+startDataService('dataServiceName')
+```
```

Comment:
Did you also want to add the pre-existing API methods `setTileLayer`, `removeTileLayer`, `setGeojsonLayer`?